homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -178,11 +177,10 @@ def put(self, request, username, entity_number):
         # Make sure the entity actually supports brightness
         entity_features = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
 
-        if (entity_features &
-                SUPPORT_BRIGHTNESS &
-                (entity.domain == "light")) == SUPPORT_BRIGHTNESS:
-            if brightness is not None:
-                data[ATTR_BRIGHTNESS] = brightness
+        if entity.domain == "light":
+            if (entity_features & SUPPORT_BRIGHTNESS) == SUPPORT_BRIGHTNESS:
```

Comment:
I had that thought going through this, but I wasn't sure I wanted to tackle it in this PR. I suppose that since I'm touching it though I should clean it up. 😄